src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const rentCar = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return rentCar - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return rentCar - SHORT_TERM_DISCOUNT;
```

Comment:
Instead of using `else if`, you can simply use an `if` statement here. Since the previous `if` block contains a `return`, the code execution will not reach this point if the condition is met.